README.md

```diff
@@ -542,14 +542,17 @@ bot type: `floor`
 
 ![image](https://user-images.githubusercontent.com/7338312/148694081-9e90f75d-dcef-4a68-b98a-11c37b2b616a.png)
 
-Track the floor price of an NFT collection on OpenSea or Solanart.
+Track the floor price of an NFT collection on OpenSea, Solanart, Solsea or MagicEden
 
 ### Bot Configuration
 
 ```json
 {
-  "marketplace": "opensea",                         # string: one of: opensea, solsea or solanart
+  "marketplace": "opensea",                         # string: one of: opensea, solanart, solsea or magiceden
   "name": "ethereum",                               # string: collection name/id from source
+  "color": true,                                    # bool/OPTIONAL: requires nickname
+  "arrows": true,                                   # bool/OPTIONAL: add price arrows
```

Comment:
@rssnyder Maybe we should replace this with the `decorator` argument. I initially added this arrow argument since I saw no need to change the decorator with the floor price. However, maybe other people need this feature, and it keeps it consistent with the rest of our codebase. What do you think?

Reply:
I only added the decorator feature as a customer wanted it. If you have no need for it here then I don't see a reason to add it. Unless you just want to spend more time in the code :)